### PR TITLE
Adding Cocoon DY180363

### PR DESCRIPTION
**Pull request by PHeonix25**
Both minified & complete configurations are required - the complete configuration won't work to initially flash the device.

### CODE REVIEW

File: src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
+---
+title: Cocoon DY180363 RGBW 8W Bulb
+date-published: 2024-10-06
+type: light
+standard: au, uk, global
```
Comment (seuros) [inline src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md]: it should be just global

Comment (jesserockz) [reply src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md]:
```suggestion
standard: global
```

File: src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+logger:
+
+# Connectivity configuration
+ota:
+  - platform: esphome
+    password: !secret ota_pass
+wifi:
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password
+  fast_connect: True
```
Comment (jesserockz) [inline src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md]:
```suggestion
  ap:

```

File: src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+esp8266:
+  board: esp01_1m
+
+# Enable logging
+logger:
+
+# Connectivity configuration
+ota:
+  - platform: esphome
+    password: !secret ota_pass
```
Comment (jesserockz) [inline src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md]:
```suggestion

```

File: src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+#   username: !secret mqtt_user
+#   password: !secret mqtt_pass
+#   birth_message:
+#   shutdown_message:
+#   will_message:
+network:
+  # enable_ipv6: True # Uncomment if you're using IPv6
+ota:
+  - platform: esphome
+    password: !secret ota_pass
```
Comment (jesserockz) [inline src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md]:
```suggestion

```

File: src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+update:
+wifi:
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password
+  fast_connect: True
```
Comment (jesserockz) [inline src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md]:
```suggestion
  ap:
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

File: src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md [hunk 6]
```diff
@@ -0,0 +1,289 @@
+---
+title: Cocoon DY180363 RGBW 8W Bulb
+date-published: 2024-10-06
```
Comment (jesserockz) [inline src/docs/devices/Cocoon-DY180363-RGBW-8W-Bulb/index.md]:
```suggestion
date-published: 2024-10-09
```

**Review by jesserockz — APPROVED** [review]

**Comment by PHeonix25** [comment]
Thank you both!

**Pull request merged**